controllers/prize_controller.go

```diff
@@ -0,0 +1,34 @@
+package controllers
+
+import (
+	"net/http"
+
+	"github.com/drift-org/backend/models"
+	"github.com/gin-gonic/gin"
+	"github.com/kamva/mgm/v3"
+)
+
+type PrizeController interface {
+	Create(context *gin.Context)
+}
+
+type prizeController struct{}
+
+func NewPrizeController() PrizeController {
+	return &prizeController{}
+}
+
+func (ctrl *prizeController) Create(context *gin.Context) {
```

Comment:
Do we want to check for uniqueness of prizes? Presumably, we wouldn't want them to add a duplicate prize by accident?

Reply:
I don't think so. As of now, we don't really have a plan as to what format the name/description fields will be. For things like coupon codes, sure maybe it makes sense, but what about other types of prizes that don't have a 'digital format' yet (ex: piece of merch), for those we'd expect name/description to be the same.

My guess is later on we'll add some field which we can use to validate

Reply:
Ok, I think we should add another field (because we should display something if someone adds a duplicate prize by accident). Let's do this in a separate ticket. Can you make an issue?